### PR TITLE
ci-operator: format user output and debug log differently

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Originally I thought this was going to be really hard to do, and it does
seem to kind of break the architecture of logrus since each entry has an
embedded formatter, but it works, so ... ?

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
(This puts JSON into the debug/admin log and keeps normal text for users)

File: cmd/ci-operator/main.go [hunk 1]
```diff
@@ -263,9 +262,10 @@ func setupLogger() (*secrets.DynamicCensor, io.Closer, error) {
 	censor := secrets.NewDynamicCensor()
 	logrus.SetFormatter(logrusutil.NewFormatterWithCensor(logrus.StandardLogger().Formatter, &censor))
 	logrus.SetOutput(ioutil.Discard)
-	logrus.AddHook(&writer.Hook{
-		Writer: os.Stdout,
-		LogLevels: []logrus.Level{
+	logrus.AddHook(&formattingHook{
```
Comment (droslean) [inline cmd/ci-operator/main.go:265]: That won't respect our censor formatter and we will end up leaking a secret. Can you check this or write a test that ensures that?

Comment (stevekuznetsov) [reply cmd/ci-operator/main.go:265]: Good point ...

Comment (droslean) [reply cmd/ci-operator/main.go:265]: I remember I hit that issue before and I opened https://github.com/sirupsen/logrus/issues/1153. It worth taking a look at it.

Comment (stevekuznetsov) [reply cmd/ci-operator/main.go:265]: I don't think *that* particular bug will hit me, since I'm forcefully formatting in this code :|

**Comment by stevekuznetsov** [comment]
I'll just pull this one in - it's a formatting/style change and everyone's out for the day anyway.

**Pull request merged**